### PR TITLE
Name of the PR

### PR DESCRIPTION
**Pull request by mariusbaciu**
Description of the PR

### CODE REVIEW

File: 1.txt [hunk 1]
```diff
@@ -8,4 +8,7 @@
 7
 1000
 line 1
-line 2
\ No newline at end of file
+line 2
+pr 1
```
Comment (MihaiMariusBaciu) [inline 1.txt]: Pr Commet 1

File: 1.txt [hunk 2]
```diff
@@ -8,4 +8,7 @@
...
+pr 2
```
Comment (MihaiMariusBaciu) [inline 1.txt]: Pr Comment 2

File: 1.txt [hunk 3]
```diff
@@ -8,4 +8,7 @@
...
+pr 3
```
Comment (MihaiMariusBaciu) [inline 1.txt]: PR Comment 3

**Comment by MihaiMariusBaciu** [comment]
Please Implement The Changes and than push to check again

**Comment by mariusbaciu** [comment]
Ok, will  be pushin in around 30 minutes

**Comment by MihaiMariusBaciu** [comment]
This is Ok Now Please Rebase and make a single Commit

**Pull request merged**